src/WebJobs.Script.Grpc/Server/DefaultHttpProxyService.cs

```diff
@@ -56,13 +59,23 @@ public async Task EnsureSuccessfulForwardingAsync(ScriptInvocationContext contex
 
             if (httpProxyTaskResult is not ForwarderError.None)
             {
+                _logger.LogDebug($"The function invocation failed to proxy the request with ForwarderError: {httpProxyTaskResult}");
```

Comment:
Was this in the original PR? We shouldn't be using string interpolation in log messages.

Reply:
Ah crap I totally missed this, and yes it was - I will fix it in both branches

Reply:
Also there are a bunch of string interpolation in log messages across the host that we should probably fix at some point too.